Wrappers.h

```diff
@@ -1,67 +1,44 @@
+#include "Party.h"
+#include "GuitarStore.h"
+
```

Comment:
add the missing `#pragma once`
don't include any header files.
forward declare all types that are being wrapped
- [ ] revise